web/client-api/src/main/java/io/deephaven/web/client/api/tree/JsTreeTable.java

```diff
@@ -248,13 +248,14 @@ public int depth() {
                 return depthColumn[offsetInSnapshot];
             }
 
-            public void appendKeyData(Object[][] keyTableData, boolean expanded) {
+            public void appendKeyData(Object[][] keyTableData, boolean expanded, boolean expandAll) {
```

Comment:
I'd probably change this to just take the action directly instead of expanded/expandAll.

Reply:
Agreed, both these ternaries could be shifted instead to the only callsite.